eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/ManagementOutputLibrary.cs

```diff
@@ -29,6 +29,9 @@ public class ManagementOutputLibrary : AzureOutputLibrary
         private PageableWrapperProvider? _asyncPageableWrapper;
         internal PageableWrapperProvider AsyncPageableWrapper => _asyncPageableWrapper ??= new PageableWrapperProvider(true);
 
+        private ProviderConstantsProvider? _providerConstants;
+        internal ProviderConstantsProvider ProviderConstants => _providerConstants ??= new ProviderConstantsProvider();
```

Comment:
so actually this way might be more clear but semantically it is different:
```suggestion
        internal ProviderConstantsProvider ProviderConstants {get;}= new ProviderConstantsProvider();
```
in this way, the value is created once this type instance is created.
The way in the PR, the value is created once this propery is called.

Feel free to ignore this.